docs/docs/node-exemption

```diff
@@ -0,0 +1,22 @@
+---
+title: Node Exemption
+---
+
+## Exempting Nodes from the Eraser Pipeline
+
+When deploying Eraser, you can specify whether there is a list of nodes you would like to `include` or `exclude` from the cleanup process using the `--filter-nodes` argument.  
```

Comment:
do we want to add this to the exclusion doc? https://azure.github.io/eraser/docs/exclusion

Reply:
since filter-nodes are added with v0.3, let's mention that